packages/istanbul-reports/lib/text/index.js

```diff
@@ -52,6 +51,28 @@ function formatPct(pct, width) {
     return fill(pct, width || PCT_COLS, true, 0);
 }
 
+function nodeMissing(node) {
+    const metrics = node.getCoverageSummary();
```

Comment:
We should short-circuit this function with an early return for summary lines:
```js
if (node.isSummary()) {
  return '';
}
```

This will simplify the remaining code.

Reply:
Done.